packages/npm/prefs/android/src/main/java/com/candlefinance/prefs/PrefsPackage.kt

```diff
@@ -0,0 +1,17 @@
+package com.candlefinance.prefs
+
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.uimanager.ViewManager
+
+
+class PrefsPackage : ReactPackage {
+  override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+    return listOf(Prefs(reactContext))
+  }
```

Comment:
purely stylistic but I like this syntax better :)
```suggestion
  override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> = listOf(Prefs(reactContext))
```

Reply:
Please do haha